### ISSUE TITLE
Tests fail for perl < 5.30

### ISSUE THREAD

**Issue by eserte** [youpong/Data-Gimei#1]
The test suite fails for older perls: http://matrix.cpantesters.org/?dist=Data-Gimei%20v0.0.4

The test failure is:
```
#   Failed test 'use Data::Gimei;'
#   at t/00_compile.t line 4.
#     Tried to use 'Data::Gimei'.
#     Error:  Feature bundle "5.30" is not supported by Perl 5.20.1 at /home/cpansand/.cpan/build/2022011519/Data-Gimei-v0.0.4-v9kyrc/blib/lib/Data/Gimei/Name.pm line 4.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2022011519/Data-Gimei-v0.0.4-v9kyrc/blib/lib/Data/Gimei/Name.pm line 4.
# Compilation failed in require at /home/cpansand/.cpan/build/2022011519/Data-Gimei-v0.0.4-v9kyrc/blib/lib/Data/Gimei.pm line 5.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2022011519/Data-Gimei-v0.0.4-v9kyrc/blib/lib/Data/Gimei.pm line 5.
# Compilation failed in require at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.
# Looks like you failed 1 test of 1.
t/00_compile.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

If it's really intended that this module does not work for older perl, then it should be specified already in Makefile.PL/Build.PL (e.g. by setting `MIN_PERL_VERSION`). This would avoid unnecessary fail reports.

**Comment by youpong** [comment]
The Perl version got specified in cpanfile.
The test suite passes: http://matrix.cpantesters.org/?dist=Data-Gimei+v0.0.5

**Comment by youpong** [comment]
I think this is fixed in v0.0.5. If we run into again, we can re-open this issue.
